rio/robot.py

```diff
@@ -80,6 +92,53 @@ def _simulationPeriodic(self) -> None:
     def testPeriodic(self) -> None:
         """Called every time during test mode"""
 
+    def extraInfo(self) -> None:
+        """
+        Called by a TimedRobot timer, contianing data that should not change much
+        but should be kept current on networktables or other connectors.
+
+        This is kind of a catchall/misc method, runs every 2 or so seconds.
+        """
+
+        data = self.getDeployData()
+
+        for key in data:
+            key_entry = self.build_table.getEntry(str(key))
+            key_entry.setString(str(data[key]))
+
+    def getDeployData(self):
+        """
+        Returns appropriate deploy data.
+        """
+
+        fakedata = {
+            "git-hash": "0000000",
+            "git-branch": "sim/sim",
+            "build-host": "SimulatedLaptop",
+            "builder": "SimUser",
+            "path": "/sim/simulatedrobot",
+            "build-date": "never",
+        }
+
+        print(deployinfo.getDeployData())
+
+        if RobotBase.isReal():
+            try:
+                with open("/home/lvuser/py/deploy.json") as fp:
+                    return json.load(fp)
+            except FileNotFoundError:
+                return fakedata
+        else:
+            return fakedata
```

Comment:
It might be useful to see this information during the sim, even if not all of it might be available. At the very least I could see catching myself using the wrong branch or something with that.

Reply:
As of right now the deploy.json is only built at deploytime, but i bet i could add a little extra local data to handle providing at least the current branch locally :3